### PR TITLE
v0.10: update to Cilium v1.10.11

### PR DESCRIPTION
**Pull request by tklauser**
Update the image digests and install Cilium v1.10.11 by default.

### CODE REVIEW

File: .github/cilium-cli-test-job-chart/values.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 
 tag: latest
-cilium_version: v0.10.10
+cilium_version: ""
```
Comment (sayboras) [inline .github/cilium-cli-test-job-chart/values.yaml:6]: I can't find why this is updated in commit message, can you clarify ?

Comment (tklauser) [reply .github/cilium-cli-test-job-chart/values.yaml:6]: It's set from the github actions explicitly, no need to specify the default value here as well.

**Review by twpayne — APPROVED** [review]

**Review by sayboras — APPROVED** [review]
:shipit:

**Comment by tklauser** [comment]
Closing in favor of #862

**Pull request closed without merging**